eng/targets/Helix.Common.props

```diff
@@ -18,23 +18,22 @@
     <HelixAvailableTargetQueue Include="Windows.81.Amd64.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Windows.10.Amd64.EnterpriseRS3.ASPNET.Open" Platform="Windows" EnableByDefault="false" />
```

Comment:
We test on this plus Windows.10.Amd64.ClientRS4.VS2017.Open. Should I upgrade to more recent Windows.10 builds❔

Reply:
Seems like we should try to leave the matrix alone on older releases shouldn't we?  Otherwise we might just create extra test failures/differences to investigate across branches?  Like maybe the idea would be we only upgrade when required, like with 16.04 being EOL

Reply:
@HaoK overall, are you suggesting I leave the Windows queues alone and do _not_ add OSX 11.00 or Ubuntu 20.04 but clean up or upgrade from CentOS 7, Debian 8, Fedora 28, OSX 10.13, and Ubuntu 16.04❔

(I would pick Ubuntu 18.04 over 20.4 at the moment because it's more commonly used. EOL is 2 years in the future.)

Reply:
Correct, I'd basically only update 16.04 -> 18.04 and just drop Centos7/Debian8/Fedora28.  If we don't have to update OSX 1013 -> 1014, or any windows versions, I wouldn't do either.

Given that we aren't working actively in this branch, minimizing churn with the queues seems better than being proactive to update things (assuming things are mostly working on the current queues).

Reply:
What's the oldest Win10 version officially supported by dotnet? We should at least test the oldest and newest.

Reply:
> What's the oldest Win10 version officially supported by dotnet?

@Pilchie @halter73 is there a Windows 10 or OSX support matrix for our servicing releases somewhere❔